alexandra/app.py

```diff
@@ -62,14 +61,28 @@ def dispatch_request(self, body):
         elif req_type == 'IntentRequest':
             intent = body['request']['intent']['name']
             intent_fn = self.intent_map.get(intent, self.unknown_intent_fn)
+            slots = body['request']['intent'].get('slots', {})
+
+            # Python 2.7 compatibility.
+            try:
+                slot_list = slots.iteritems()
+            except AttributeError:
+                slot_list = slots.items()
```

Comment:
let's just do .items() then instead of the try/except. dictionary wont be big enough for the copy to matter